### PR TITLE
New Alcatraz Gear

### PR DESCRIPTION
**Pull request by Kurfursten**
I love the warden's spare secway key as a joke item but it's never gonna be a good product. But what if it came bundled with your own special secway? Stun helmet is a nod to our favorite armless secoff.

**Stun Helmet**
Projects orange light
Fends off psychic threats (none of the other benefits of tinfoil like anti-teleport, anti-tracking, or anti-HUD)
Works like a baton you can use with bite intent
Charge in recharger or by standing on mech charger

**Custom Secway Kit**
Customize your own secway! By default it has 2x health the normal secway and is much better equipped to ram people without knocking yourself over; it comes with the warden's spare secway key. You can also choose 3 upgrades (12 options)
image

Includes 6 designs to choose from!

image
image

**Bullet Tree**
A beautiful metal tree that will grow over time. Its fruits can be fed to the ammolathe for metal. You can't use it with other lathes and it's useless without an ammolathe.

🆑 
* tweak: Increases the count of items per Alcatraz crate to 7 (from 6)
* rscdel: You can no longer buy individual pax implants. The warden's spare secway key is no longer in the crates.
* rscadd: Alcatraz crates now include the 5-pack of pax implants, Stun Helmet, Custom Secway Kit, and Bullet Tree.

### CODE REVIEW

File: code/game/objects/structures/vehicles/secway.dm [hunk 1]
```diff
@@ -58,3 +59,228 @@
...
+	if(baby.can_take_pai == FALSE)
+		to_chat(user,"<span class='info'>With an unprinted circuitboard you could slot in a personal AI.</span>")
+	if(!baby.impact_sound)
+		to_chat(user,"<span class='info'>There's a good spot for a hailer to make legal noises when you hit stuff.</span>")
+	if(!baby.can_spacemove)
+		to_chat(user,"<span class='info'>A spacepod core is just what this thing needs to move in space.</span>")
+	if(baby.movement_delay > 0.9)
+		to_chat(user,"<span class='info'>If it had an RTG cell, this thing could go a little faster.</span>")
+	if(baby.explodes)
+		to_chat(user,"<span class='info'>With an energy gun, it could be rigged to fire a countermeasure if destroyed up.</span>")
```
Comment (Sooose) [inline code/game/objects/structures/vehicles/secway.dm]: The grammar at the end is a little confusing. Does it proc when it's destroyed or shot up?

Comment (Kurfursten) [reply code/game/objects/structures/vehicles/secway.dm]: When destroyed, it's a typo

> imagine writing feedback messages at 3am

Comment (Sooose) [reply code/game/objects/structures/vehicles/secway.dm]: I can't sleep and I saw an interesting new PR. Help.

Comment (Kurfursten) [reply code/game/objects/structures/vehicles/secway.dm]: I actually meant myself, the line you (correctly) pointed out was wrong is a feedback message.

We've both lost control of our lives

**Comment by DrSnips** [comment]
The bullet tree is really underwhelming since metal is literally everywhere in the form of chairs or 100s of sheets just laying around the station.
Maybe it provides a variety of other materials too?

**Comment by SonixApache** [comment]
why doesn't the bullet tree just grow bullets
stick a mag in, bullets grow in it

**Comment by Kurfursten** [comment]
If the bullet tree seems underwhelming I think the main reason is because I just thought it'd be nice as warden to have a little bonsai to take care of. More about the pleasure of having some decor.

@DrSnips Perhaps it could give plasma too. That would let you print a lot of the special shells like dragon's breath. The only other materials ammolathe wants are in gun recipes.

@SonixApache This is a cute idea but it requires you to have the mag first, it's less straightforward to use, and it cuts out the middleman instead of encouraging people to use the ammolathe (which feels underused)

**Comment by calzilla1** [comment]
Maybe have the tree's fruit give metal + a random other material that the lathe uses (Plastic, silver, glass, plasma, maybe a rare chance of gold)

**Comment by Blithering** [comment]
Knife and ectoplasm make ghetto syndiechair? Not sure we should encourage sec to turn their rides into killmobiles.

**Comment by Kurfursten** [comment]
Ectoplasm would make the knife (and plastic bumper upgrade) useless because it would no longer bump people. The custom secway only does effects when it bumps things, **not** when it runs over them, unlike MULE/syndie chair. It can't hurt people who are laying down and it doesn't stun whoever it bumps (crawling is permitted).

Ectoplasm is also somewhat rare. I expect to see it almost never because you don't get to slap on upgrades as you find them, you have to commit to your 3 upgrades when you build it, a one time deal. So there has to be a source of ectoplasm like a wizard, *and* a trader selling things, *and* a security officer with access to both.

**That said**, the knife is too much of a meme option I am almost certainly replacing anyway since in my initial testing it just seems like it would be miserable to keep stabbing people by accident in the hallways. It's a really pitiful amount of damage so it's really going annoy pedestrians more than it successfully deals with antags. Less dangerous than an armed roomba.

**Comment by Blithering** [comment]
I was just worried officers would slap a knife on always since that's something they have by default.

**Comment by Kurfursten** [comment]
Yeah, that's just about the only positive the idea had.

**Comment by calzilla1** [comment]
"You can no longer buy individual pax implants." Something something unatomic

**Comment by Kurfursten** [comment]
Oh, I don't think I made it clear in the changelog (probably because I revised it a couple of times).

Pax Implants are going into a 5-pack where you get all 5 at once instead of buying them individually, as a random possibility in the Alcatraz crates. The thing with Pax Implants is that it's rare to see the marketed, so I am hoping that if you get them bundled with other sec goodies you might have more incentive to sell them.

**Comment by drawsstuff** [comment]
> Pax Implants are going into a 5-pack where you get all 5 at once instead of buying them individually, as a random possibility in the Alcatraz crates. The thing with Pax Implants is that it's rare to see the marketed, so I am hoping that if you get them bundled with other sec goodies you might have more incentive to sell them.

I play both trader and security regularly and ask that you not, namely because 150 is a steep investment when I'm up to no good as a traitor trader. That and when I'm not, it's cheap price of them that makes them so accessible for both traders and security. The current "Trader meta" is to just buy all the bundle items like the wardrobe anyways, I'd rather not see feature loss.

**Comment by Kurfursten** [comment]
If you buy all the bundle items anyway, you'll be getting all of them for free as part of the bundle.

I think bundles are good because otherwise traders just end up being an intermediary between the vending machine and the end user [what I mean by this is people just come up and ask you to buy something for them from the vending machine]. There's no incentive to buy something **before** you know there's a customer unless you gain information [like why people buy the WW immediately], hence why everybody just moves the vendor onto their ship. Putting things into bundles does a few very good things for play
* When traders buy one product, they get a bunch of other things that they now have to market to turn a profit on. So you might be getting the alcatraz crate to try to resell a HECU mask, but now you have all the other things that you have some incentive to market.
* Bundles encourage traders to buy right away to find out what their stock actually is, instead of leaving it in the vending machine until they secure a buyer. This means the products are actually out on the table where people can see them. This is good for everybody.
* Bundles reduce clutter on the vendor itself. On live, there are currently 43(!!) lines of products in the tradevend. That's great for variety but it's getting to the point where I (and I assume everyone else) just uses ctrl+f

I would very much like to move most of the products into bundles. Livestock crates into one, flatpacks into another, miscellany into another. In order to do that though I am going to need to finish a few more products. It would be great if every department had a bundle as attractive as Alcatraz is for Security.

**Comment by Eneocho** [comment]
> 
> @SonixApache This is a cute idea but it requires you to have the mag first, it's less straightforward to use, and it cuts out the middleman instead of encouraging people to use the ammolathe (which feels underused)

Take the gun, place next to the tree, it drops an appropiate magazine to the same tile.

**Comment by Kurfursten** [comment]
@Eneocho Hilarious but never worth the security risk

---

Big update here. Fixes a lot of things. Canned the knife upgrade entirely. Electrode countermeasure now only requires a taser. Added the horse design. Started work on the ammo tree but it needs spritework, I am in communication with spriters on this.

**Comment by Blithering** [comment]
Soon?

**Review by ShiftyRail — APPROVED** [review]

**Pull request merged**